PyRDF/Node.py

```diff
@@ -178,6 +225,385 @@ def graph_prune(self):
         return self.is_prunable()
 
 
+class FriendInfo(object):
+    """
+    A simple class to hold information about friend trees.
+
+    Attributes:
+        friend_names (list): A list with the names of the `ROOT.TTree` objects
+            which are friends of the main `ROOT.TTree`.
+
+        friend_file_names (list): A list with the paths to the files
+            corresponding to the trees in the `friend_names` attribute. Each
+            element of `friend_names` can correspond to multiple file names.
+    """
+
+    def __init__(self, friend_names=[], friend_file_names=[]):
+        """
+        Create an instance of FriendInfo
+
+        Args:
+            friend_names (list): A list containing the treenames of the friend
+                trees.
+
+            friend_file_names (list): A list containing the file names
+                corresponding to a given treename in friend_names. Each
+                treename can correspond to multiple file names.
+        """
+        self.friend_names = friend_names
+        self.friend_file_names = friend_file_names
+
+    def __bool__(self):
```

Comment:
Do we really need to redefine `__bool__`?

Also, is it necessary to check both lists to know there are no friends?

Reply:
The redefinition of `__bool__` was there because we wanted to check that both lists were non-empty. This is not necessary but probably just an extra check that we can retrieve both the name of the friend tree and its files. Although this makes the recreation of the friend tree on the distributed worker easier because we're sure the files exist (?)